modules/smart-agent_kubernetes-volumes/detectors-kubernetes-volumes.tf

```diff
@@ -36,6 +36,8 @@ EOF
     parameterized_subject = var.message_subject == "" ? local.rule_subject : var.message_subject
     parameterized_body    = var.message_body == "" ? local.rule_body : var.message_body
   }
+
+  max_delay = var.volume_space_max_delay
```

Comment:
Not directly related to this PR, is it intentional to have no heartbeat detector for Kubernetes volumes metrics?

Reply:
I can answer for this one. Indeed, there is no "regular" metric reporting so it is basically not possible, as for lambda for example

Reply:
I don't know why I did not create this one. It could be a bit duplicated of `kubernetes-common` of course but it remains a different source of metric which can break independently so it seems relevant. I will create an issue.